src/context/auth/authContext/AuthContext.types.ts

```diff
@@ -0,0 +1,4 @@
+export type AuthContextValuesTypes = {
+  isLoggedIn: boolean;
```

Comment:
Jedna drobna rzecz - jak to się nazywa `AuthContext`, to może użyjemy tutaj `isAuth`/`isAuthenticated`?
Swoją drogą, jaka jest różnica pomiędzy "authorized" i "authenticated"?